examples/python/custom_processing.py

```diff
@@ -123,8 +128,8 @@ def main():
 
     # +get images:
     images = []
-    for _ in range(batch_size):
```

Comment:
this is potentially dangerous. If there are a lot of images, it could be a very large request. For now, the safest thing to do would be to use the batch size to create groups of images for requests. e.g. 20 images with a batch size of 4 -> 5 requests

Reply:
@varunsh-xilinx  How does batch sizing work? What is default if not specified, and what are the practical limits?